### DISCUSSION TITLE
Issue configuring followtag

### DISCUSSION THREAD

**Discussion by truxnell** [renovatebot/renovate#12583] [Request Help]
I'm trying to incorporate a game server into my kurbenetes config.
The goal was to use renovate to manage my game server tags.

However, this is the first docker source I have where I dont want to follow 'latest.  With their tag scheme, 'latest' is sometimes experimental releases, and 'stable' is what is declared stable by the Devs, and is what Steam auto-updates to.
I have hunted around as I expected this would be pretty standard/basic, so i added "followTag: stable" (and commented out matchUpdateTypes.

```
// Game servers
{
  matchDatasources: ["docker"],
  automerge: true,
  automergeType: "branch",
  requiredStatusChecks: null,
  respectLatest: false,  // Added this in to try and force not to use latest, possibly unnessecary?
  followTag: "stable", // Follow 'stable' tag release
  //matchUpdateTypes: ["major","minor", "patch", "digest"], // Commented out as I cant have this with followTag
  matchPackageNames: [
  "factoriotools/factorio"
  ]
},
```

However, this stops this package updating, and throws a error that it cant find the stable tag:

```
{
  "packageFile": "apps/base/games/factorio-formic/factorio-formic.yaml",
  "deps": [
    {
      "depName": "factoriotools/factorio",
      "currentValue": "1.1.42",
      "datasource": "docker",
      "replaceString": "1.1.42",
      "versioning": "docker",
      "autoReplaceStringTemplate": "{{newValue}}{{#if newDigest}}@{{newDigest}}{{/if}}",
      "depIndex": 0,
      "updates": [],
      "warnings": [
        {
          "topic": "factoriotools/factorio",
          "message": "Can't find version with tag stable for factoriotools/factorio"
        }
      ]
    },
}
```

Tag is definitely present and I cant spot anything else in my renovate config that would stop it working.
I did notice higher up in my log that it is still only grabbing the latest label when refreshing labels.

```
DEBUG: getLabels(https://index.docker.io, factoriotools/factorio, latest)
```

Thanks for having a look!

**Sources:**
 
**My repo:** https://github.com/Truxnell/home-cluster
**Renovate config:** (needs a tidy) https://github.com/Truxnell/home-cluster/blob/main/.github/renovate.json5
**Example demo file that should update:** https://github.com/Truxnell/home-cluster/blob/main/apps/base/games/factorio-formic/factorio-formic.yaml
**Dockerhub:** https://hub.docker.com/r/factoriotools/factorio

**Comment by viceice** [comment]
`followTag` only works for npm datasource.

**Reply by rarkins** [reply]
Yes. Renovate treats Docker tags as values/versions already without any special config, so remove `followTag`. What's an example docker image you have, and what update do you expect?
